src/main/java/com/commercetools/sync/producttypes/utils/ProductTypeUpdateLocalizedEnumActionUtils.java

```diff
@@ -0,0 +1,133 @@
+package com.commercetools.sync.producttypes.utils;
+
+import com.commercetools.sync.commons.exceptions.DuplicateKeyException;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.LocalizedEnumValue;
+import io.sphere.sdk.producttypes.ProductType;
+import io.sphere.sdk.producttypes.commands.updateactions.AddLocalizedEnumValue;
+import io.sphere.sdk.producttypes.commands.updateactions.ChangeLocalizedEnumValueOrder;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.commercetools.sync.producttypes.utils.ProductTypeUpdateEnumActionsUtils.buildRemoveEnumValuesUpdateActions;
+import static com.commercetools.sync.producttypes.utils.ProductTypeUpdateEnumActionsUtils.buildMatchingEnumValuesUpdateActions;
+import static com.commercetools.sync.producttypes.utils.ProductTypeUpdateEnumActionsUtils.buildAddEnumValuesUpdateActions;
+import static com.commercetools.sync.producttypes.utils.ProductTypeUpdateEnumActionsUtils.buildChangeEnumValuesOrderUpdateAction;
+import static java.util.Collections.emptyList;
+
+public final class ProductTypeUpdateLocalizedEnumActionUtils {
```

Comment:
The `ProductTypeUpdateLocalizedEnumActionUtils` and `ProductTypeUpdatePlainEnumActionUtils` classes are similar in the `buildUpdateAction` method and someone could argue that I could also reuse the code in the `ProductTypeUpdateEnumActionsUtils` class, the same as it's done with other methods.

I did it that way first but the code ended up more tight and difficult to understand since all three particular methods for adding, removing and change order of the enums have to be passed as parameters in the common method placed at ProductTypeUpdateEnumActionsUtils, making this method to have 6 parameters where some of them are BiFunction, TriFunction...  (long and "complex" signature)

So I finally decided to repeat some code in `ProductTypeUpdateLocalizedEnumActionUtils` and `ProductTypeUpdateLocalizedEnumActionUtils` in order to clarify and decouple the `buildUpdateAction` method of both classes.